scripts/testnet.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+set -e #u
+
+if [ -z "$GCLOUD_INCLUDED" ]; then
+  source scripts/gcloud.sh 
+fi
+
+MIN_FUNDS=0.01291
+HOPRD_ARGS="--data='/app/db/ethereum/testnet/bootstrap' --password='$BS_PASSWORD'"
+ZONE="--zone=europe-west6-a"
+
+# $1 = role (ie. bootstrap, node-4)
+# $2 = network name
+vm_name() {
+  echo "$2-$1"
+}
+
+# $1 = vm name
+disk_name() {
+  echo "$1-dsk"
+}
+
+# $1=account (hex)
+balance() {
+  ethers eval "new ethers.providers.JsonRpcProvider('$RPC').getBalance('$1').then(b => formatEther(b))"
+}
+
+# $1=account (hex)
+fund_if_empty() {
+  echo "Checking balance of $1"
+  local BALANCE="$(balance $1)"
+  echo "Balance is $BALANCE"
+  if [ "$BALANCE" = '0.0' ]; then
+    echo "Funding account ... $RPC -> $1 $MIN_FUNDS"
+    ethers send --rpc "$RPC" --account "$FUNDING_PRIV_KEY" "$1" $MIN_FUNDS --yes
+    sleep 60
+  fi
+}
+
+# $1 = IP
+get_eth_address(){
+  echo $(curl $1:3001/api/v1/address/eth)
```

Comment:
As I'm afraid this will break at some point whenever we update this endpoint in the feature, I already see some of these commands being ported to `nodejs` scripts that share constants across our repos.

Reply:
we shouldn't change API endpoints. They _are_ the constants.